openedx/core/djangoapps/credit/api.py

```diff
@@ -111,6 +111,33 @@ def get_credit_requirements(course_key, namespace=None):
     ]
 
 
+def get_credit_requirement(course_key, namespace, location):
+    """ Returns the requirement of a given course, namespace and name
+
+    Args:
+        course_key(CourseKey): The identifier for course
+        namespace(str): Namespace of requirement
+        name: Name of the requirement
+
+    Returns:
+        Credit Requirement object
+    """
+    return CreditRequirement.get_course_requirement(course_key, namespace, location)
+
+
+def set_credit_requirement_status(user_name, requirement, status="satisfied", reason=None):
+    """Update Credit Requirement Status for given username and requirement
+        if exists else add new
+
+    Args:
+        user_name(str): username of the user
+        requirement(CreditRequirement): CreditRequirement object
+        status(str): status of the requirement
+    """
+    CreditRequirementStatus.add_or_update_requirement_status(
```

Comment:
This format should be:
```python
results = my_object.some_method(
    arg1,
    arg2,
    arg3,
)
```
https://github.com/edx/edx-platform/wiki/Python-Guidelines#breaking-long-lines